packages/dnote/build.sh

```diff
@@ -0,0 +1,55 @@
+TERMUX_PKG_HOMEPAGE=https://www.getdnote.com/
+TERMUX_PKG_DESCRIPTION="A simple command line notebook for programmers"
+TERMUX_PKG_LICENSE="AGPL-V3,GPL-3.0"
+TERMUX_PKG_MAINTAINER="2096779623 <admin@utermux.dev>"
+TERMUX_PKG_VERSION=2.0.1
+TERMUX_PKG_SRCURL=https://github.com/dnote/dnote/archive/refs/tags/server-v${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=28cc3bf93b3849b9a4a5e65e531f8a34d6be6048427d924c56ab8c7887676bad
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_AUTO_UPDATE=true
+TERMUX_PKG_DEPENDS="dnote-server,dnote-client,postgresql"
```

Comment:
Probably dependencies here can be removed, dnote-server and dnote-client depend on the mother package automatically, which is what we want

Reply:
Does it really automatically depend, but why @MrAdityaAlok asked me to add subpackages to depends?

https://github.com/termux/termux-packages/pull/10797#pullrequestreview-983569815

Reply:
[subpackages depend on the mother package automatically](https://github.com/termux/termux-packages/blob/master/scripts/build/termux_create_debian_subpackages.sh#L77), unless `TERMUX_SUBPKG_DEPEND_ON_PARENT` is set to false (or really anything other than `unversioned` or `deps`). 

The mother package normally does not depend on the subpackages though. Typical situation is that we have mother package `libfoo` with library, and subpackage `foo` which contains a program linked against `libfoo` (and so the subpackage depends on the mother package).

I would guess (but am not familiar with frp) that it makes more sense to have `TERMUX_PKG_SUGGESTS="frpc, frps"` for frp

Reply:
https://github.com/termux/termux-packages/actions/runs/3366734259/jobs/5583566310

same result.